scm/driver/harness/util.go

```diff
@@ -32,6 +33,30 @@ func buildHarnessURI(account, organization, project, repo string) (uri string) {
 	return repo
 }
 
+func getRepoAndQueryParams(slug string) (string, string, error) {
+	params := url.Values{}
+	s := strings.TrimSuffix(slug, "/+")
+	splitSlug := strings.Split(s, "/")
+	params.Set("accountIdentifier", splitSlug[0])
+	params.Set("routingId", splitSlug[0])
```

Comment:
we don't need `routingId` if we set `accountIdentifier`, right? or is it just to be sure?

Reply:
just to be sure.